flags/application.json

```diff
@@ -14,6 +14,26 @@
     "shift": 4,
     "undocumented": true
   },
+  "RPC_PRIVATE_BETA": {
+    "description": "Application has access to the RPC HTTP server",
+    "shift": 5,
+    "undocumented": true
+  },
+  "ALLOW_ASSETS": {
+    "description": "Relates to RPC application assets",
+    "shift": 8,
+    "undocumented": true
+  },
+  "ALLOW_ACTIVITY_ACTION_SPECTATE": {
+    "description": "Whether the application can allow users to spectate a game from RPC",
+    "shift": 9,
+    "undocumented": true
+  },
+  "ALLOW_ACTIVITY_ACTION_JOIN_REQUEST": {
+    "description": "Allows the application can allow users to request to join a user's game from RPC",
```

Comment:
Poor wording - possibly:
`"Enables activity join requests for the application",`